qml/pages/AddProject.qml

```diff
@@ -344,6 +344,9 @@ Dialog {
             }
 
             Component.onCompleted: {
+                if (!projectId) {
+                    projectId = db.getUniqueId().toString()
```

Comment:
I'm not sure this is a correct approach, since `saveProject` returns a new UUID when a new project is saved:

`projectId = db.saveProject(values);`

Unless this line in `Database.cpp` does not work as expected: 
```
QString id = values["id"].isNull() ? getUniqueId().toString() : values["id"].toString();
```
https://github.com/olpeh/wht/blob/master/src/Database.cpp#L476

Reply:
Oh, might be I did not read your whole description. The above only works correctly if simply creating a new project and not adding tasks. I'll have to test this a bit.